src/klein/_app.py

```diff
@@ -73,6 +73,9 @@ def __call__(
         """
 
 
+KleinQueryValue = Union[str, int, float]
```

Comment:
I see that the available types are documented here...and I think they are implemented via werkzeug routing.

So maybe add a comment that these should be based on https://werkzeug.palletsprojects.com/en/1.0.x/routing/#built-in-converters.

But I don't know what other types could be added in the future .... 

So maybe it's ok without any comment :)

Reply:
I used a list I saw in the Klein docs. Personally, I think anything other than string is weird and just convert myself if need be, so I don't use this feature at all. YYMV.